tests/common_test.go

```diff
@@ -111,11 +119,22 @@ func init() {
 		log.Fatal(err)
 	}
 
-	keys, err := server.NewKeystoreFromDir("testdata", LoadKey)
-	if err != nil {
-		log.Fatal(err)
+	if os.Getenv("TESTHSM") == "" {
```

Comment:
Does this need to be mutually exclusive? How about writing it as:
```go
keys, err := server.NewKeystoreFromDir("testdata", LoadKey)
if err != nil {
	log.Fatal(err)
}
if os.Getenv("TESTHSM") != "" {
	if err := keys.AddFromURI(params.RSAURI, LoadURI); err != nil {
		log.Fatal(err)
	}
	if err := keys.AddFromURI(params.ECDSAURI, LoadURI); err != nil {
		log.Fatal(err)
	}
}
s.SetKeystore(keys)
```

Reply:
One returns `Keystore` and the other returns `DefaultKeystore`, so this seemed the easiest.

Reply:
Well how about this then:
```go
keys := server.NewDefaultKeystore()
if err := keys.AddFromDir("testdata", LoadKey); err != nil {
	log.Fatal(err)
}
if os.Getenv("TESTHSM") != "" {
	if err := keys.AddFromURI(params.RSAURI, LoadURI); err != nil {
		log.Fatal(err)
	}
	if err := keys.AddFromURI(params.ECDSAURI, LoadURI); err != nil {
		log.Fatal(err)
	}
}
s.SetKeystore(keys)
```

Reply:
What goal does this serve? The keys in those URI's are the exact keys in that directory.